### PR TITLE
Fix wut release script

### PR DESCRIPTION
**Pull request by jonathanadler**

### CODE REVIEW

File: packages/wix-ui-tpa/package.json [hunk 1]
```diff
@@ -61,7 +61,7 @@
     "update-components": ".wuf/update-components.sh",
     "pr-release": "# SRC_MD5=`/usr/bin/python $SCRIPT_DIR/md5Calc.py .` npm-ci publish temp-publish",
     "release-candidate": "node scripts/rc.js",
-    "changelog": "conventional-changelog --preset angular -i CHANGELOG.md -s",
+    "changelog": "conventional-changelog --preset angular -l wix-ui-tpa -t WUT-v -i CHANGELOG.md -s --commit-path=.",
```
Comment (argshook) [inline packages/wix-ui-tpa/package.json:64]: should it be `WUT-v` in here?

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6728

File: packages/wix-ui-tpa/package.json (hunk 1 above)
Comment (jonathanadler) [reply packages/wix-ui-tpa/package.json:64]: hmm 🤔 ?
Isn't that what's written here?

Comment (argshook) [reply packages/wix-ui-tpa/package.json:64]: yep, i meant if it's needed in there at all?

Comment (jonathanadler) [reply packages/wix-ui-tpa/package.json:64]: yes, it's needed for the library to detect the previous release by its tag

**Pull request merged**